File: python/src/Behavioral/Visitor/BookInfo.py
# //BookInfo.java - one of three concrete Visitees
from AbstractTitleInfo import AbstractTitleInfo

class BookInfo (AbstractTitleInfo) :

	author = None

	def __init__(self,titleName,author) :
		super(BookInfo,self).__init__(titleName)
		self.author = author

	def accept (self,titleBlurbVisitor) :
		titleBlurbVisitor.visit(self)

File: ruby/src/Behavioral/Visitor/BookInfo.rb
# //BookInfo - one of three concrete Visitees
#require "AbstractTitleInfo"

class BookInfo < AbstractTitleInfo

	attr_accessor :author

	def initialize(titleName,author)
 		super(titleName)
		@author = author
	end

	def accept (titleBlurbVisitor)
 		titleBlurbVisitor.visit(self)
	end

end

